core/templates/dev/head/pages/creator_dashboard/creator_dashboard.html

```diff
@@ -376,7 +376,365 @@ <h2 class="activity-title protractor-test-exp-summary-tile-title">
       </ul>
     </div>
   </div>
+  <style>
+    .oppia-creator-dashboard-main-content {
+      padding: 0;
+      width: 100%;
+    }
+    .oppia-creator-dashboard-main-content .stats-card {
+      min-height: 63px;
+      position: relative;
+    }
+    .oppia-creator-dashboard-main-content .stats-card .stat-description {
+      line-height: 1.646;
+      margin-bottom: 3em;
+      padding-right: 10px;
+    }
+    .oppia-creator-dashboard-main-content .stats-card .stat-value-with-rating {
+      top: 0.75em;
+      position: absolute;
+    }
+   .oppia-creator-dashboard-main-content .stats-card .stat-value-without-rating {
+      bottom: -0.7em;
+      position: absolute;
+    }
+    .oppia-creator-dashboard-mobile-mode {
+      margin: auto;
+      width: 100%;
+    }
+    .oppia-dashboard-tabs {
```

Comment:
Some of these have different prefixes -- oppia-dashboard and oppia-creator-dashboard, for example. We should standardize on a naming convention.

However, I think it's fine to at least just get the correct rules in the right place first. We can then enforce naming conventions in a separate pass.

One question: have you confirmed that all CSS classes here are used at the top-level of the creator dashboard, and not in subcomponents of that page? We need to ensure that CSS classes for directives that are part of a larger page go within that directive's HTML instead.

Reply:
Yes, at the moment, I have only moved the CSS selectors at one place. I will put up a naming convention in the document itself.
Regarding your question, all CSS selectors in `create_dashboard.html` are used only by it. CSS selectors for `create_activity_modal_directive.html` and `upload_activity_modal_directive.html` were already present in the corresponding files.

Reply:
That sounds great. Thanks!